http_check/README.md

```diff
@@ -1,32 +1,102 @@
-# Http_check Integration
+# HTTP Integration
 
-## Overview
+# Overview
 
-Get metrics from http_check service in real time to:
+Monitor the up/down status of local or remote HTTP endpoints. The HTTP check can detect bad response codes (e.g. 404), identify soon-to-expire SSL certificates, search responses for specific text, and much more. The check also submits HTTP response times as a metric.
 
-* Visualize and monitor http_check states
-* Be notified about http_check failovers and events.
+# Installation
 
-## Installation
+The HTTP check is packaged with the Agent, so simply [install the Agent](https://app.datadoghq.com/account/settings#agent) on any host from which you want to probe your HTTP sites. Though many metrics-oriented checks are best run on the same host(s) as the monitored service, you may want to run this status-oriented check from hosts that do not run the monitored sites.
 
-Install the `dd-check-http_check` package manually or with your favorite configuration manager
+If you need the newest version of the HTTP check, install the `dd-check-http` package; this package's check will override the one packaged with the Agent. See the [integrations-core](https://github.com/DataDog/integrations-core#installing-the-integrations) repository for more details.
```

Comment:
@gmmeyer do you know yet if `dd-check-http` will indeed be the package name? Since this check is called `http_check`, I suppose our build process may call it `dd-check-http-check`, but perhaps we can override that stuttering name? :)

(Same question for `dns_check`, `tcp_check`, `snmp_check`)